playbooks/roles/ocp-install/templates/create_oauth.j2

```diff
@@ -0,0 +1,13 @@
+---
+apiVersion: config.openshift.io/v1
+kind: OAuth
+metadata:
+  name: cluster
+spec:
+  identityProviders:
+  - name: my_identity_provider
+    mappingMethod: claim
+    type: HTPasswd
+    htpasswd:
+      fileData:
+        name: htpass-secret
```

Comment:
Is this the file expected from the upstream terraform repo? If yes then we can just take username and password as input vars from the user for cluster-admin role in terraform itself and then create this file in the playbook.

Reply:
Yes, it comes from the ocp4-upi-powervm repo. By doing it this way I don't need to enter password in a variable file (which I really don't like) or putting a script which asks for password before the kick-off.